en/security.rst

```diff
@@ -10,6 +10,53 @@ This chapter describes the CUBRID security features. CUBRID provides Packet Encr
 Packet Encryption
 =================
 
+The need for packet encryption
+------------------------------
+Without an encrypted connection between the client and the server, a hacker with access to the network can monitor all traffic and steal and exploit data exchanged between the client and server. In this way, a third party (middle-man) steals and make bad use  information, it is called a man in the middle (MITM) attack. This MITM attack can be prevented by adding a secure authentication procedure to the connection between the client and server.
+
+Packet encryption method
+------------------------------
+CUBRID uses SSL/TLS (Secure Socket Layer/Transport Layer Security) protocol to encrypt data transmitted between the client and server. The CUBRID server uses OpenSSL for encryption, and the client can make an encrypted connection using JDBC or CCI Driver. The encryption protocols supported between the client and server are SSLv3, TLSv1, TLSv1.1, and TLSv1.2.
+
+.. note:: **SSL/TLS**
+
+	SSL was first developed by Netscape as an encryption protocol that provides authentication and data encryption between a client and server working over a network. Nescape released version 3.0, which improved security flaws in 1996. The SSL 3.0 version became the basis for TLS 1.0, and was defined by the IETF in January 1999 as the `RFC2246 <https://tools.ietf.org/html/RFC2246>`_ standard. The last update is `RFC5246 <https://tools.ietf.org/html/RFC5246>`_ . TLS is defined based on SSL 3.0, it is almost similar to SSL 3.0.
+
+	SSL/TLS provides Server Authentication, Client Authentication and Data Encryption functions. Authentication refers to a procedure to verify that the other party is correct, and encryption refers to preventing access to contents even if data is stolen.
+
+
+Server setup for packet encryption
+------------------------------------
+**Set encryption mode and non-encryption mode**
+
+CUBRID can set either an encryption mode or a non-encryption mode, and the default is non-encryption mode. To change to the encryption mode, you can set the encryption mode by changing the SSL parameter value in cubrid_broker.conf. If you change the SSL parameter value of cubrid_broker.conf, you must restart the broker. For detailed configuration method, refer to :ref:`broker-configuration`\ .
```

Comment:
```suggestion
CUBRID can set either an encryption mode or a non-encryption mode, and the default is set to non-encryption mode. To change to the encryption mode, you can set the encryption mode by changing the SSL parameter value in cubrid_broker.conf. If you change the SSL parameter value of cubrid_broker.conf, you must restart the broker. For detailed configuration method, refer to :ref:`broker-configuration`\ .
```

Reply:
CUBRID brokers can be configured either an encryption mode or a non-encryption mode, and the default mode is non-encryption mode. To change to the encryption mode, you can set the encryption mode by changing the SSL parameter value in cubrid_broker.conf. If you change the SSL parameter value of cubrid_broker.conf, you must restart the broker. For detailed configuration method, refer to :ref:`broker-configuration`\ .

Reply:
I'll keep it current.